adyenv6core/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -833,19 +836,37 @@ public void initializeCheckoutData(Model model) {
 
         Optional<PaymentMethod> sepaDirectDebit = alternativePaymentMethods.stream().
                                                                             filter(paymentMethod -> ! paymentMethod.getType().isEmpty() &&
-                                                                                    PAYMENT_METHOD_SEPA_DIRECTDEBIT.contains(paymentMethod.getType())).findFirst()
-                ;
+                                                                                    PAYMENT_METHOD_SEPA_DIRECTDEBIT.contains(paymentMethod.getType())).findFirst();
+
         if(sepaDirectDebit.isPresent())
         {
             model.addAttribute(PAYMENT_METHOD_SEPA_DIRECTDEBIT, true);
         }
 
+        baseStore = baseStoreService.getCurrentBaseStore();
+
+        //Verify allowedCards
+        String cardsName = null;
```

Comment:
we are already passing allowedcards to card component. is there any benefit of doing it here?

Reply:
It's not needed for localization, but I added as a way to verify if the allowedCards set on hybris side are also configured and allowed on adyen side.